src/main/resources/db/migration/V2_20__rename_visit_room.sql

```diff
@@ -7,13 +7,11 @@ UPDATE legacy_data
 
 ALTER TABLE visit ALTER COLUMN visit_room drop not null;
 
-
 UPDATE visit
     SET visit_room = NULL
         FROM (SELECT v.id,v.visit_room FROM visit v JOIN legacy_data ld ON ld.visit_id = v.id) AS subquery
-    WHERE visit.id = subquery.id;
-
+    WHERE visit.id = subquery.id and visit_start < CURRENT_DATE;
```

Comment:
Not really part of ticket but found a defect in sql, this should only set visit_room for historic legacy visits